scenarios/all_in_one/playbooks/04_all_in_one_specific.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Prepare the system for the all_in_one_scenario
+  hosts: localhost
+  gather_facts: false
+  become: true
+  vars:
+    virt_packages:
+      - python3
+      - libvirt
+      - virt-install
+      - qemu-kvm
+      - virt-manager
+      - python3-pip
+      - python3-lxml
+      - python3-libvirt
+  tasks:
+    - block:
+      - name: "Install virt-manager"
+        package:
+          name: "{{ virt_packages }}"
+          state: present
+
+      - name: Start libvirtd
+        service:
+          name: libvirtd
+          state: started
+
+      - name: Add user to qemu.conf
+        lineinfile:
+          path: /etc/libvirt/qemu.conf
+          line: 'user = "redhat"'
+
+      - name: Add group to qemu.conf
+        lineinfile:
+          path: /etc/libvirt/qemu.conf
```

Comment:
This shouldn't be required anymore

Reply:
Great, I will made a commit to remove that part